File: edudoc/src/app/model/interfaces/custom/district-progress-report.dto.ts
import { IEntity } from "@mt-ng2/base-service";

export interface IDistrictProgressReportDto extends IEntity {
    Id: number,
    ProviderId: number,
    FirstName: string,
    LastName: string,
    ServiceAreaId: number,
    ServiceAreaName: string,
    TotalIEPStudents: number,
    TotalEncounters: number,
    TotalCompletedReports: number,
}

File: edudoc/src/Model/DTOs/DistrictProgressReportDto.cs
using System.Data.SqlClient;
using System.Data.SqlClient;
using System;
using System.Collections.Generic;

namespace Model.DTOs
{
    public class DistrictProgressReportDto
    {
        public int ProviderId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int ServiceAreaId { get; set; }
        public string ServiceAreaName { get; set; }
        public int TotalIEPStudents { get; set; }
        public int TotalEncounters { get; set; }
        public int TotalCompletedReports { get; set; }
    }

}
